sparkplug/config/connection.py

```diff
@@ -141,6 +148,8 @@ def pump(self, connection, channel):
         timeout = connection.heartbeat * 0.4 or None
         while True:
             _log.debug("Waiting for a message.")
+            if self.should_terminate:
```

Comment:
While this _can_ use `self` to access `should_terminate` I feel like using `AMQPConnector.should_terminate` will be more consistent

Reply:
Agree, changed